installer/uninstall-cloudflow.sh

```diff
@@ -67,15 +67,47 @@ if [[ $REPLY =~ ^[Yy]$ ]]; then
     kubectl delete crd alertmanagers.monitoring.coreos.com \
       prometheuses.monitoring.coreos.com \
       prometheusrules.monitoring.coreos.com \
+      sparkapplications.sparkoperator.k8s.io \
       scheduledsparkapplications.sparkoperator.k8s.io \
+      kafkausers.kafka.strimzi.io \
+      kafkatopics.kafka.strimzi.io \
+      kafkas.kafka.strimzi.io \
+      kafkamirrormakers.kafka.strimzi.io \
+      kafkaconnects2is.kafka.strimzi.io \
+      kafkaconnects.kafka.strimzi.io \
+      kafkaconnectors.kafka.strimzi.io \
+      kafkabridges.kafka.strimzi.io \
       servicemonitors.monitoring.coreos.com \
-      sparkapplications.sparkoperator.k8s.io \
       flinkapplications.flink.k8s.io \
       cloudflowapplications.cloudflow.lightbend.com \
       cloudflowapplications.lightbend.com \
       --ignore-not-found=true
 
-    echo "Done !"
+    echo "Removing ClusterRoles..."
+    kubectl delete clusterrole cloudflow-nfs-fdp-nfs \
+      cloudflow-flink-flink-operator \
+      cloudflow-sparkoperator-cr \
+      strimzi-cluster-operator-global \
+      strimzi-cluster-operator-namespaced \
+      strimzi-entity-operator \
+      strimzi-kafka-broker \
+      strimzi-topic-operator \
+      --ignore-not-found=true
+
+    echo "Removing ClusterRoleBindings..."
+    kubectl delete clusterrolebinding cloudflow-nfs-fdp-nfs \
+      cloudflow-flink-flink-operator \
+      cloudflow-sparkoperator-crb \
+      cloudflow-operator-bindings \
+      strimzi-cluster-operator \
+      strimzi-cluster-operator-kafka-broker-delegation \
+      --ignore-not-found=true
+
+    echo "Removing StorageClasses..."
+    kubectl delete sc nfs-client \
```

Comment:
This should be removed only if nsf is used, what if efs is used?

Reply:
If NFS is not used (i.e. storage class `nfs-client` is not present), this command won't do anything. It won't stop the script with an error.

Reply:
In general it is a good practice not to make unnecessary calls. Also we need to cleanup all storage classes.

Reply:
This is necessary because when NFS is used, uninstalling cloudflow without deleting the storage class will create an error next time cloudflow is installed.